cmd/cluster/dr/update_dr.go

```diff
@@ -0,0 +1,117 @@
+// Licensed to Yugabyte, Inc. under one or more contributor license
+// agreements. See the NOTICE file distributed with this work for
+// additional information regarding copyright ownership. Yugabyte
+// licenses this file to you under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package dr
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	ybmAuthClient "github.com/yugabyte/ybm-cli/internal/client"
+	"github.com/yugabyte/ybm-cli/internal/formatter"
+	ybmclient "github.com/yugabyte/yugabytedb-managed-go-client-internal"
+)
+
+var updateDrCmd = &cobra.Command{
+	Use:   "update",
+	Short: "Update DR for a cluster",
+	Long:  `Update DR for a cluster`,
+	Run: func(cmd *cobra.Command, args []string) {
+		authApi, err := ybmAuthClient.NewAuthApiClient()
+		if err != nil {
+			logrus.Fatalf(ybmAuthClient.GetApiErrorDetails(err))
+		}
+		authApi.GetInfo("", "")
+
+		drName, _ := cmd.Flags().GetString("dr-name")
+		databases, _ := cmd.Flags().GetStringArray("databases")
+		clusterId, err := authApi.GetClusterIdByName(ClusterName)
+		if err != nil {
+			logrus.Fatalf("Could not get cluster data: %s", ybmAuthClient.GetApiErrorDetails(err))
+		}
+		drId, err := authApi.GetDrIdByName(clusterId, drName)
+		if err != nil {
+			logrus.Fatal(err)
+		}
+		namespacesResp, r, err := authApi.GetClusterNamespaces(clusterId).Execute()
+		if err != nil {
+			logrus.Debugf("Full HTTP response: %v", r)
+			logrus.Fatalf(ybmAuthClient.GetApiErrorDetails(err))
+		}
+		dbNameToIdMap := map[string]string{}
+		for _, namespace := range namespacesResp.Data {
+			dbNameToIdMap[namespace.GetName()] = namespace.GetId()
+		}
+		databaseIds := []string{}
+		for _, databaseString := range databases {
+			for _, database := range strings.Split(databaseString, ",") {
+				if databaseId, exists := dbNameToIdMap[database]; exists {
+					databaseIds = append(databaseIds, databaseId)
+				} else {
+					logrus.Fatalf("The database %s doesn't exist", database)
```

Comment:
IIUC we're effectively duplicating the API level checks here, do we need this?

Reply:
Yes, we would like to validate and fail in the CLI itself.

Reply:
But why the need for this additional validation? The API call will fail synchronously anyway

Reply:
Well, actually, we need the above API call to fetch database IDs from names, hence it is not another wasted API call, we are just checking an in-memory map.

Reply:
Ah it's translation code, my bad